### ISSUE TITLE
loss remains a constant training on my own custom dataset

### ISSUE THREAD

**Issue by Kongsea** [JimLee4530/RCNN#1]
I modified this repository to train on my own custom dataset.

The input image size of my dataset is much larger than yours and it's 224*224. I tried using your network structure or add more RCL layers and max pooling layers to make the features smaller to input to global max pooling layers, however, after training some iterations (maybe less than 100 iterations), the loss became a constant and did not change at all.

I have tried to change the learning rate to both larger and smaller, but it remains a constant.

Any suggestions or advice? Thank you.

**Comment by JimLee4530** [comment]
In my opinion, you can change T(time step) in every RCL layers, like from 3 to 0, to check RCL layers is exactly work in your experiment.
And you also can add you kernel size in every convolution layer. In the paper, RCL is used for capture the context in lower layers , and my image size is 32*32 ,so my kernel size is 3, but your image size is 224*224. I think you can change larger one, like 11.
I'm so sorry for respond to you after a week.

**Comment by jyning12** [comment]
did you mean the T(time step) is the differ between the add layers' s two different layer, I hava difficult in reading that paper

**Comment by JimLee4530** [comment]
I think you can change the T in different experiment, but it should be same between the add layer.
